### ISSUE TITLE
Parser error when class has multiple methods without access modifier

### ISSUE THREAD

**Issue by gregorej** [gregorej/vala-intellij-plugin#17]
Sample:

```Vala
class Foo {
	int method1 () {
	}

	string method2 () {
	}

	void method3 () {
	}
}
```

**Comment by romalytvynenko** [comment]
```vala
namespace Test {
	class MyClass {
		public void test() {

		}
		void test_m() {
		
		}
	}
}
```
This case contains parser error too.

**Comment by gregorej** [comment]
Fixed, as of 3dba901c05248fa95f4f5f9d4c0addf7e6e990f7
